CromIAM/src/main/scala/cromiam/webservice/CromIamApiService.scala

```diff
@@ -30,14 +32,18 @@ trait SwaggerService extends SwaggerUiResourceHttpService {
 trait CromIamApiService extends RequestSupport
   with EngineRouteSupport
   with SubmissionSupport
-  with QuerySupport {
+  with QuerySupport
+  with CromIamInstrumentation {
 
   implicit val system: ActorSystem
   implicit def executor: ExecutionContextExecutor
   implicit val materializer: ActorMaterializer
 
   protected def configuration: CromIamServerConfig
 
+  val config = ConfigFactory.load()
```

Comment:
Instead of loading this again, perhaps add it as a member to the `CromIamServerConfig`.

Something like:
```scala
final case class CromIamServerConfig
(
  rootConfig: Config, // <-- new
  cromIamConfig: CromIamConfig,
  cromwellConfig: ServiceConfig,
  cromwellAbortConfig: ServiceConfig,
  samConfig: ServiceConfig,
  swaggerOauthConfig: SwaggerOauthConfig
)
```

Reply:
Instead of putting it inside `CromIamServerConfig`, I added it inside `CromIamServer` object where it is being loaded the first time.